Tests/LibWeb/Text/expected/css/hsl-with-number-percentage-calc.txt

```diff
@@ -0,0 +1 @@
+rgb(88, 101, 242)
```

Comment:
It's more a question than a real comment, but shouldn't the actual result of the  serialization be `hsl(235 85.6% ...`?

I know that we don't support serialization of color entirely. Still that would be nice to make this test independent of the (lack of this) feature.

Reply:
Surprisingly no, this is actually correct! All colours that are RGB internally serialise like this. https://www.w3.org/TR/css-color-4/#serializing-sRGB-values